src/qilib/utils/storage/mongo.py

```diff
@@ -52,6 +52,138 @@ def transform_bson(self, value: Any) -> Any:
         return value
 
 
+def _encode_int(value: int) -> str:
```

Comment:
TBH, I don't like the list of methods within a file that contains classes. IMHO, I would rather prefer a static class

Reply:
Maybe, create a static encoder/decoder class with private string and int encoders/decoders?